packages/components/input/src/input.vue

```diff
@@ -45,6 +45,8 @@
           v-bind="attrs"
           :type="showPassword ? (passwordVisible ? 'text' : 'password') : type"
           :disabled="inputDisabled"
+          :formatter="formatter"
```

Comment:
Line 48/49: What are the formatter and parse props on input element for? They're not native attributes for input element.